### DISCUSSION TITLE
A alguien mas no le aparecen los grupos en TEAMS?

### DISCUSSION THREAD

**Discussion by JOSHU411** [Launch-X-Latam/MisionFrontEnd#437] [Team Posts]
Que fue lo que hicieron para ingresar a los grupos en teams?
Apenas ayer me llego el correo con mi cuenta y contraseña y entre a github para iniciar el onboarding, pero al entrar a teams veo que no hay nada

**Comment by jossuema** [comment]
Hola compañeros lo que tienen que hacer es completer el onboarding, hay un formulario que deben contestar luego de eso les llegara un correo a la cuenta de innovacion donde esta la invitacion al team de launch x, ingresan y luego hay que esperar a que la invitacion sea aceptada. Yo estoy esperando la aceptacion.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: ***@***.***>
Sent: Friday, October 7, 2022 2:04 PM
Cc: ***@***.***>
Subject: [Launch-X-Latam/mision-frontend] A alguien mas no le aparecen los grupos en TEAMS? (#6)


Que fue lo que hicieron para ingresar a los grupos en teams?
Apenas ayer me llego el correo con mi cuenta y contraseña y entre a github para iniciar el onboarding, pero al entrar a teams veo que no hay nada

**Comment by JOSHU411** [comment]
ah ok, muchas gracias, creí que se aceptaba en automático.
